test.js

```diff
@@ -1,374 +1,361 @@
 'use strict';
 const path = require('path');
-const assert = require('assert');
+const test = require('ava');
 const gutil = require('gulp-util');
 const rev = require('./');
```

Comment:
You can use `import` statements here and drop the use strict statement. AVA transpiles test files with Babel.